### ISSUE TITLE
Error cannot find module jest

### ISSUE THREAD

**Issue by 4ureli1** [mhlabs/cfn-diagram#7]
After installing cfn-dia, while running the program I got the error below:
```
cfn-dia
internal/modules/cjs/loader.js:968
  throw err;
  ^

Error: Cannot find module 'jest'
Require stack:
- /usr/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js
- /usr/lib/node_modules/@mhlabs/cfn-diagram/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/usr/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js:5:27)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/lib/node_modules/@mhlabs/cfn-diagram/graph/Vis.js',
    '/usr/lib/node_modules/@mhlabs/cfn-diagram/index.js'
  ]
}
```

It's not very bad to correct. Just install jest: `npm install -g jest`. it should be added in the project dependencies

**Comment by ljacobsson** [comment]
Thanks for reporting. :+1: 

Fixing straight away

**Comment by ljacobsson** [comment]
`jest` was already there as a `devDependency`. For some reason it had been included as a `require` in a non-test file.

Fixed in v1.1.1

Thanks again for reporting

**Comment by 4ureli1** [comment]
thanks
